__test__/components/__snapshots__/tabs.test.js.snap

```diff
@@ -102,7 +102,7 @@ Array [
             <h5
               className="black"
             >
-              28/2/2019
+              29/2/2019
```

Comment:
Same issue here. Seems similar to the class name issue. This is definitely not scalable.